### PR TITLE
Feat: Adds Student buckets

### LINKED ISSUE #57: [FEATURE] Add a student bucket to contributor stats
### Is your feature request related to a problem?

We have a number of students contributing to OpenSearch, see https://astanatimes.com/2023/03/kazakhstans-top-students-to-participate-in-amazons-flagship-project/.

### What solution would you like?

Add a "student" bucket of users into https://github.com/opensearch-project/project-tools/blob/main/lib/github/buckets.rb and allow for a students.txt to sort.

### PR DESCRIPTION
**Pull request by ashwin-pc**
### Description
Adds a new bucket for student contributors

### Issues Resolved
Closes #57 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: lib/github/buckets.rb [hunk 1]
```diff
@@ -23,11 +27,11 @@ def all
     end
 
     def all_members
-      members.concat(contractors)
+      members.to_a + contractors.to_a
```
Comment (dblock) [inline lib/github/buckets.rb:30]: You can just do `members + contractors`, these are already arrays.

**Review by dblock — APPROVED** [review]
Looks good. We really need to be adding tests here too, maybe you could start filling up https://github.com/opensearch-project/project-tools/blob/main/spec/github/pull_requests_spec.rb, but I won't hold the PR for this.

Let's add something about students to README, e.g. in https://github.com/opensearch-project/project-tools#contributor-stats.

File: lib/github/buckets.rb (hunk 1 above)
Comment (ashwin-pc) [reply lib/github/buckets.rb:30]: @dblock They arent arrays, they are Hashs. Atleast in [`contributors.rb`](https://github.com/opensearch-project/project-tools/blob/main/lib/github/contributors.rb#L41) they are. When i run pr stats, it fails unless they are explicitly converted to arrays

Comment (dblock) [reply lib/github/buckets.rb:30]: Oh but that is a problem, you get back an array. We need to merge hashes and get back a hash, don't we?

Write a test for this, and you'll see what works.

**Comment by dblock** [comment]
I merged for now, we can iterate on this further.

**Comment by dblock** [comment]
Added students bucket info in https://github.com/opensearch-project/project-tools/pull/60.

**Pull request merged**